### DISCUSSION TITLE
在微博看见个相声讲给大家听

### DISCUSSION THREAD

**Discussion by Winter-And-You-Gone** [The-Run-Philosophy-Organization/run#520] [General]
甲：今天我给大家说段相声，相声名字叫……
乙：哎哎哎，还说什么相声啊，听说了吗，唐山烧烤摊打人了！
甲：知道啊，不是抓起来了吗？
乙：抓起来就行了啊？你这要求忒低了吧。这一打人，我上街都没安全感了。
甲：那您现在这怎么出来溜达了？
乙：我这不是打算run吗？看看这个国家出了多少事，现在吃个烧烤都要担惊受怕，没希望了，我得run到国外去！
甲：run到国外就能解决问题？
乙：那肯定，国外是法制国家，比中国好多了，没这毛病。
甲：我不太信。
乙：你不信，那咱们来场比赛。
甲：什么比赛？
乙：咱俩分别说一件时事，如果中国做的不好，那给外国加一分，外国做的不好，给中国加一分，做的好的就各自加分。最后看看总得分谁更高。如果外国得分高，那我就得run到外国去。
甲：也行，那您开始吧。
乙：唐山烧烤摊打人。
甲：嗯，这是坏事。
乙：对，所以得给外国加一分。
甲：现在1:0。
乙：徐州小花梅案，轰动全国！
甲：也是坏事。
乙：这就2:0了。
甲：那美国每年上万起枪击案，这次中国得加1分吧？
乙：那不对。
甲：为啥不对呢？
乙：你是哪国人？
甲：中国人啊。
乙：你是中国人，为什么要关心外国的事情呢？外国做的好坏，和你有一毛钱关系吗？难道不该关注我们身边的事情吗？
甲：你这么说，听上去好像也挺有道理。
乙：是啊，所以这一分不能加。
甲：那现在的比分？
乙：还是2:0。
甲：那中国精准扶贫，惠及数千万人口，可以加1分吧？
乙：还是不能加。
甲：这又是为什么？
乙：你是哪国人？
甲：怎么又问这个问题啊，刚才说了，中国人。
乙：你如果真的是个爱国的中国人，就应该知道，政府本来就应该服务于纳税人，所以说无论他做了什么工作，这都是应该的，我们不需要有任何感恩。爱国就要指出这个国家的错误，而不是歌功颂德，批评而不是表扬，这才是真爱国。
甲：你这么说，听上去好像也挺有道理。
乙：所以这一分也不能加，还是2:0。
甲：我说了两个都不作数，那您说一个吧。
乙：美国新冠肺炎应对得当，应该给外国加一分。
甲：哎等会等会，不是死了100万吗？为啥给外国加分，不是应该给中国加分吗？
乙：至少人家敢报道出来，对不对？
甲：报道出来他也死了啊。
乙：对于自由世界的人民而言，早就享受过生活了，根本就是看淡生死，他们不能忍受的，是被防疫和核酸检测剥夺了自由。再说这病毒本来就是个小感冒，人家国家政府是民选的，万万不能强迫老百姓戴口罩。就算有几个人死了，但是他们也自由了。
甲：所以这应该加分？
乙：得加，若为自由故，二者皆可抛嘛。
甲：那就3:0？
乙：不对，是4:0。
甲：怎么还给外国加了两分呢？
乙：中国防疫看上去是少死了人，但是代价是什么呢？付出了经济的代价，显然是失败的，这也算1分。
甲：那外国经济也不比中国好啊。
乙：但是人家敢报道出来啊，这又是一件好事，你要在说我把这一分也得加上，那就得5:0……
甲：那中国也保护了生命，也是成绩吧。
乙：不对，你看人家美国，死100万人，国家一点事没有，这说明什么呢？过去我们总说美国人怕死人，现在才发现，人家美国根本不怕死人，这要是打起仗来，美国人嗷嗷往前冲，他不怕死啊，你这么怕死怎么办呢，就只能打输了。
甲：那就6:0了……
乙：中国人说做核酸都去做核酸，是不是奴性？
甲：7:0。
乙：再看看人家日本面对疫情，素质很高，大家都戴口罩，都不给别人添麻烦，是不是高素质？
甲：这就8:0了。
乙：再说了，我从新闻上看到西安地铁拖拽案，保安员太暴力了，他这样做，全中国人都没有安全感，因为14亿人都可能会是下一个，9:0。
甲：那美国呢？
乙：我在美国的朋友说了，他身边昨天一个被枪击打死的都没有，安全的很，10:0
甲：那美国新闻里也有被打死的啊？
乙：你不去治安不好的区就可以了，富人区很安全，这是能自我控制的，但中国没有明显的富人区和穷人区分界线，所以每个人都同时面临危险。
甲：你说的都对，这一阵就11:0了。
乙：前几天日本前首相安倍晋三薨了，中国网民言语不敬，很没素质，12:0。
甲：那日本网络上天天骂中国啊，还年年出书呢。
乙：这说明日本人真性情，敢表达，言论自由，为什么人家日本人不骂别人就骂你呢，中国人难道不该反思吗？
甲：行了，13:0了，这么比下去，我看中国一分也得不了。
乙：这就是真实实力的对比，说明外国就是比中国强，我得run出去。
甲：那你现在run的怎么样了？
乙：嗨，别提了，人家不让我去，说我有什么……移民倾向。
甲：那你不得据理力争啊。
乙：我争了啊，我大骂中国不放我出去，耽误我前程。
甲：那是中国不放你吗？
乙：反正我是没能run出去。
甲：这你就得从自身反思原因。
乙：我为什么要反思？这都是这个国家的错，是原生家庭的错，是社会的错，是洗脑教育的错，他们都错了，只有我是纯洁无辜的。
甲：那你现在打算怎么办呢？
乙：我想过了，我得搞大翻译运动啊。让美国人看看，这个国家有多坏，美国军队一生气，就身着白盔白甲来接我了。
甲：那你有什么翻译作品了？
乙：很快就有了，不给你说了，我日语培训班要上课了。
甲：好家伙，现学啊！

**Comment by ghost** [comment]
https://twitter.com/lidangzzz/status/1560517635527286784
